docs/products/postgresql/howto/use-dblink-extension.rst

```diff
@@ -0,0 +1,121 @@
+Use the PostgreSQL ``dblink`` extension
+==========================================
+
+``dblink`` is a `PostgreSQL extension <https://www.postgresql.org/docs/current/dblink.html>`_  that allows you to connect to other PostgreSQL databases and to run arbitrary queries. 
+
+With `Foreign Data Wrappers <https://www.postgresql.org/docs/current/postgres-fdw.html>`_ (FDW) you can uniquely define a remote **foreign server** in order to access its data. The database connection details like hostnames are kept in a single place, and you only need to create once a **user mapping** storing remote connections credentials.
+
+Prerequisites
+-------------
+
+To create a ``dblink`` foreign data wrapper you need the following information about the PostgreSQL remote server:
+
+* ``TARGET_PG_HOST``: The remote database hostname
+* ``TARGET_PG_PORT``: The remote database port
+* ``TARGET_PG_USER``: The remote database user to connect
+* ``TARGET_PG_PASSWORD``: The remote database password for the ``TARGET_PG_USER``
+* ``TARGET_PG_DATABASE_NAME``: The remote database name
+
+.. Note::
+
+    If you're using Aiven for PostgreSQL as remote server, the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+
+Enable ``dblink`` extension on Aiven for PostgreSQL
+-------------------------------------------------------
+
+To enable the ``dblink`` extension on an Aiven for PostgreSQL service:
+
+* Connect to the database with the ``avnadmin`` user, the following shows how to do it with ``psql``, the service URI can be found in the `Aiven console <https://console.aiven.io/>`_ service Overview page:
+
+::
+
+    psql "postgres://avnadmin:[AVNADMIN_PWD]@[PG_HOST]:[PG_PORT]/[PG_DB_NAME]?sslmode=require"
+
+.. Tip::
+
+    If you're using Aiven for PostgreSQL as remote server, you can connect to a service with the ``avnadmin`` user with the ``avn service cli`` command with the :ref:`Aiven CLI <avn-service-cli>`.
+
+* Create the ``dblink`` extension
+
+::
+
+    CREATE EXTENSION dblink;
+
+Create a foreign data wrapper using ``dblink_fdw``
+--------------------------------------------------
+
+To create a foreign data wrapper using the ``dblink_fwd`` you need to perform the following steps:
+
+*  Connect to the database with the ``avnadmin`` user, the following shows how to do it with ``psql``, the service URI can be found in the `Aiven console <https://console.aiven.io/>`_ service Overview page:
+
+::
+
+    psql "postgres://avnadmin:[AVNADMIN_PWD]@[PG_HOST]:[PG_PORT]/[PG_DB_NAME]?sslmode=require"
+
+
+* Create an user ``user1`` that will be access the ``dblink``
```

Comment:
```suggestion
* Create a user ``user1`` that will be access the ``dblink``
```
This is english being weird - since "`user`" is pronounced as if it started with "you", it gets to act as if it started with a consonant, so it's "`a`" and not "`an`".